### PR TITLE
Add a diff seconds expression

### PR DESCRIPTION
**Pull request by sheelio**
Need this for ICDS (to calculate timespent on a form).

@czue @NoahCarnahan

### CODE REVIEW

File: corehq/apps/userreports/expressions/date_specs.py [hunk 1]
```diff
@@ -86,3 +86,20 @@ def __call__(self, item, context=None):
...
+    type = TypeProperty('diff_seconds')
+    from_expression = DefaultProperty(required=True)
+    to_expression = DefaultProperty(required=True)
+
+    def configure(self, from_expression, to_expression):
+        self.from_expression = from_expression
+        self.to_expression = to_expression
+
+    def __call__(self, item, context=None):
+        from_val = transform_datetime(self.from_expression(item, context))
```
Comment (dimagimon) [inline corehq/apps/userreports/expressions/date_specs.py]: F821 undefined name 'transform_datetime'

File: corehq/apps/userreports/expressions/date_specs.py [hunk 2]
```diff
@@ -86,3 +86,20 @@ def __call__(self, item, context=None):
...
+        to_val = transform_datetime(self.to_expression(item, context))
```
Comment (dimagimon) [inline corehq/apps/userreports/expressions/date_specs.py]: F821 undefined name 'transform_datetime'

**Comment by czue** [comment]
:+1: looks good to me. not merging due to label

**Comment by sheelio** [comment]
Removing label, good to merge. 

@czue @NoahCarnahan

**Comment by snopoke** [comment]
@czue @sheelio FYI this was already possible with the `timedelta_to_seconds` function:
```
{
    "column_id": "duration",
    "datatype": "integer",
    "display_name": "Duration",
    "expression": {
        "type": "evaluator",
        "statement": "timedelta_to_seconds(time_end - time_start)",
        "context_variables": {
            "time_start": {
                "datatype": "datetime",
                "type": "property_path",
                "property_path": [
                    "form",
                    "meta",
                    "timeStart"
                ]
            },
            "time_end": {
                "datatype": "datetime",
                "type": "property_path",
                "property_path": [
                    "form",
                    "meta",
                    "timeEnd"
                ]
            }
        }
    },
    "type": "expression"
},
```

**Comment by czue** [comment]
huh, how bout that. @sheelio can you convert icds UCRs to this? probably easier long term to not have two ways of doing this.

**Comment by sheelio** [comment]
Sounds right to remove this.  Will add a PR to do that.

**Pull request merged**